### ISSUE TITLE
Integration with MyFitnessPal API

### ISSUE THREAD

**Issue by jazzsequence** [jazzsequence/recipe-box#12]
http://www.myfitnesspal.com/api

Allow recipes to be added to MFP Food Database.
Allow recipes to be pulled from MFP Food Database and edited.

**Comment by sambulance** [comment]
I requested access to the MFP API ages ago but didn't hear back. I wanted to build my own recipe book/meal planner/shopping list tool that synced with MFP. I think it's only really open to consumer services like Fitbit rather than little personal projects like ours :( would love to know if you hear back!

**Comment by jazzsequence** [comment]
@sambulance yeah, it's been a couple months and so far nothing. it's unfortunate because if just a single WordPress plugin got MFP API integration, that would be huge for both said plugin and MFP (import recipes from My Fitness Pal into WordPress! Export recipes from WordPress directly into My Fitness Pal!) which was what I was pitching when I requested access. ¯\_(ツ)_/¯
MFP does scrape pages for importing recipes, though, and to that end I added schema.org markup to try to make that process easier/work better (since I imagine that's what they're looking at) but even before I did that, they imported okay individually into MFP.
